Add correct version dependency.

mparker17 opened the pull request:
Support for pear was added in the php cookbook version 1.1.0.

Commit:
Add correct version dependency.

Support for pear was added in the php cookbook version 1.1.0.

patcon commented:
Sure thing, I'm all for adding more restrictive constraints! but if we're going to do it, isn't it `>= 0.99.0` when pear resource was introduced?

mparker17 commented:
The earliest release of the PHP cookbook which was tagged in Git was 1.1.0 (see https://github.com/opscode-cookbooks/php/releases). I haven't confirmed how the Chef code checks version numbers, but other cookbooks appear to reference git tag names.

patcon commented:
Any version from the community site api is valid, which goes back further than the tags :)

Commit:
Merge branch 'master' into feature-correct_php_cookbook_version_dep

Commit:
Set dependency to 0.99.0.

As pointed out by @patcon, even though no git tags exist for version
0.99.0, it's still a valid release version to depend on.

mparker17 commented:
@patcon I've added a commit to fix the dependency as you suggested.

patcon commented on metadata.rb:
```diff
@@ -5,7 +5,7 @@
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.rdoc'))
 version          "0.10.0"
-depends          "php"
+depends          "php", "~> 0.99.0"
```
I think the cautious operator would break things, since it would cap the acceptable version at `0.99.x`, right?

mparker17 commented:
Oh, wow, I didn't realize that was a caution operator.

Still, though, everything seemed to work fine in my runlists... I didn't get any problems.

Thanks for your work on this!
